src/basics.md

```diff
@@ -1266,6 +1267,34 @@ fn main() {
 }
 ```
 
+[ex-convert-datetime-timestamp]: #ex-convert-datetime-timestamp
+<a name="ex-convert-datetime-timestamp"></a>
+## Convert date to UNIX timestamp and vice versa
+[![chrono-badge]][chrono] [![cat-date-and-time-badge]][cat-date-and-time]
+
+Converts a date given by [`NaiveDate::from_ymd`] and [`NaiveTime::from_hms`]
+to [UNIX timestamp] using [`NaiveDateTime::timestamp`].
+Then it calculates what was the date after one billion seconds
+since [UNIX timestamp], using [`NaiveDateTime::from_timestamp`].
```

Comment:
> since [UNIX timestamp], 

UNIX Timestamp itself is number of seconds since UNIX Epoch to this statement should be:
"Then it calculates what was the date after one billion seconds since [UNIX Epoch]" or "since 1970-01-01 00:00:00" as was previously.

Reply:
That's right, thanks. I'll change it back then.